### PR TITLE
docs(iot): many fix and cloud twins documentation int-fix-iot

### PR DESCRIPTION
**Pull request by JulienVdG**

### CODE REVIEW

File: iot/iot-hub/how-to/provide-own-certificate-authority.mdx [hunk 1]
```diff
@@ -63,9 +63,9 @@ To change your Hub Certificate Authority, you must [disable your Hub](/iot/iot-h
 
 7. Re-enable your Hub to activate the replaced Certificate Authority. 
 
-<Message type="note">
+<Message type="important">
 
-Once the CA is uploaded, all existing devices will have their **Scaleway certificates deleted**, as they won't match the newly installed Certificate Authority. You will need to generate new certificates for those devices for them to be able to connect again.
+Once the CA is uploaded, all existing devices will have their **Scaleway certificates deleted**, as they won't match the newly installed Certificate Authority. You will need to generate new certificates on your side for those devices for them to be able to connect again.
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/how-to/provide-own-certificate-authority.mdx]: modified the last sentence to: You will need to generate new certificates on your side to be able to connect your devices again.

File: iot/iot-hub/how-to/connect-twins-to-grafana-cloud.mdx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+- You have [added a device](/iot/iot-hub/how-to/add-device)
+- You have [created a Grafana Cloud account](https://grafana.com/products/cloud/)
+
+Make sure the Device allows insecure connections so we can use the webclient.
+
+</Message>
+
+<Message type="note">
+
+In production, you should `Deny Insecure` connections to have the highest level of security.
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/how-to/connect-twins-to-grafana-cloud.mdx]: It's more for a real production system as opposed to a tutorial / testing stuff.
Maybe: "When designing a production system, you should `Deny Insecure` connections to have the highest level of security. "

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+	See [Create a Grafana Cloud API key](https://grafana.com/docs/grafana-cloud/reference/create-api-key/) for details.
+
+	</Message>
+
+3. Go to your Grafana instance: https://user-identifier.grafana.net/
+
+    where `user-identifier` is your username.
+
+4. Go to the graphite configuration panel:
+	-Select **Configuration**, then **Data sources** in the menu (A)
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:67]: thanks, fixed

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+	- Select Graphite on the list (B)
+
+	<Lightbox src="assets/grafanacloud-configuration-datasource-menu-graphite.png" alt="" />
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:70]: yes, the page has moved from one section to the other, thanks for spotting that !

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+
+5. Get the following information from the Graphite configuration panel:
+	- In the `HTTP` box, find the `URL` value and note the host part (`GRAPHITE_HOST`, `graphite-us-central1.grafana.net` in the example) (A)
+	- In the `Auth` box, find the `Basic Auth Details` and note the user (a number as your `GRAPHITE_USERID`) (B)
+
+	<Lightbox src="assets/grafanacloud-datasource-graphite-settings.png" alt="" />
+
+6. The push uri is constructed as follows:
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:78]: done

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+    }
+}
+```
+
+### Configuration document
+
+Let us consider we want to trace:
+- any write on the `state` (even when setting the same value twice)
+- changes on the `data` part of the document
+- we add a tag to those values in order to recover them easily in grafana.
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:130]: right, moved to it's own sentence

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 7]
```diff
@@ -0,0 +1,254 @@
...
+totalTime: PT15M
+steps:
+  - step: TODO
+    type: HowToStep
+    url: https://www.scaleway.com/en/docs/iot/iot-hub/how-to/connect-twins-to-grafana-cloud/
+image: /assets/images/docs/scaleway.png
+keywords: iot, smart, connected, embedded, devices, hubs, mqtt, twins, document store, document, cache, data, graphite, grafana
+---
+
+This tutorial will show how to connect Cloud Twins to a graphite instance on Grafana Cloud.
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:13]: changed the end to "(updates or modifications of its document's values)." which is more accurate, thanks for the suggestion !

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 8]
```diff
@@ -0,0 +1,254 @@
...
+	      "value": 12.5,
+	      "in error": false
+	    }
+	  }
+	}' | jq
+	```
+
+### Show the event in grafana
+
+1. Go to your grafana instance explore panel
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:224]: Thanks, I changed all grafana to Grafana except in urls

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 9]
```diff
@@ -0,0 +1,254 @@
...
+	- select Explore (A)
+	- ensure the graphite source is selected (B)
+	- click on the pen to switch to the query editor (C)
+	<Lightbox src="assets/grafanacloud-explore.png" alt="" />
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:228]: yes, moves the 4 of them ;)

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+
+2. See your data
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:230]: done

File: iot/iot-hub/concepts.mdx [hunk 11]
```diff
@@ -195,6 +195,15 @@ IoT Routes forward messages to non publish/subscribe destinations such as databa
 
 A topic indicates what information is contained in a message. The message broker uses topics to forward published messages to the right subscribers. Learn more about topics [on our blog](https://blog.scaleway.com/scaleway-iot-hub-introduction-to-mqtt-topics/).
 
+</Concept>
+
+<Concept>
+
+## Twins
+
+Cloud Twins, or simply Twins, are virtual representations of your physical assets. Technically, they are JSON document sets stored in a Hub. You can store and fetch data at any time with these documents. For more information, refer to [our documentation about Cloud Twins](/iot/iot-hub/reference-content/twins/).
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/concepts.mdx:204]: :+1:

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 12]
```diff
@@ -0,0 +1,254 @@
...
+	curl -sS -H "X-Auth-Token: $SCW_SECRET_KEY" -X PATCH $IOT_API/hubs/$HUB_ID -d '{
+		"twins_graphite_config": {
+			"push_uri": "https://'$GRAPHITE_USERID':'$GRAPHITE_TOKEN'@'$GRAPHITE_HOST'/metrics"
+			}
+		}'
+	```
+
+	The Hub is now connected to Grafana cloud.
+
+## Twin
```
[the remark this answers is not in the document]
Comment (JulienVdG) [reply iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:106]: what about "Using the Twin with Grafana" ?

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 13]
```diff
@@ -0,0 +1,255 @@
...
+- You have [added a device](/iot/iot-hub/how-to/add-device/)
+- You have [created a Grafana Cloud account](https://grafana.com/products/cloud/)
+
+ - Your Device allows insecure connections
+
+</Message>
+
+<Message type="note">
+
+In production, you should `Deny Insecure` connections to have the highest level of security.
```
Comment (ldecarvalho-doc) [inline iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:35]:
```suggestion
When designing a production system, you should `Deny Insecure` connections to have the highest level of security.
```

File: iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx [hunk 14]
```diff
@@ -0,0 +1,255 @@
...
+</Message>
+
+<Message type="requirement">
+
+- You have an account and are logged into [the Scaleway console](https://console.scaleway.com)
+- You have installed [curl](https://curl.se/) and [`jq` (json parsing tool)](https://stedolan.github.io/jq/) on your local computer
+- You have [created an IoT Hub](/iot/iot-hub/how-to/create-hub/)
+- You have [added a device](/iot/iot-hub/how-to/add-device/)
+- You have [created a Grafana Cloud account](https://grafana.com/products/cloud/)
+
```
Comment (ldecarvalho-doc) [inline iot/iot-hub/api-cli/connecting-twins-to-grafana-cloud.mdx:28]:
```suggestion
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

**Pull request merged**